### DISCUSSION TITLE
I lost my codes can i get my file project back from project that build in vercel?

### DISCUSSION THREAD

**Discussion by lawbyte** [vercel/community#7019] [Help] [answered]
### Summary

Please help me for restoring my files back, cz i lost my codes 

### Example

i lost my codes and i want to restore it from vercel project

### Steps to Reproduce

my problem is losing my codes and i want to restore it back

**Comment by swarnava** [comment, marked as answer]
Hi

We only store build files, not source code. So you will not be able to export them. Thanks!
